vendor/github.com/nishanths/fullstory/go.mod

```diff
@@ -1 +0,0 @@
-module github.com/fullstorydev/hauser/vendor/github.com/nishanths/fullstory
```

Comment:
You could have actually undone many of these. In my original PR, I had to add numerous `go.mod` files to make Go modules happy with the replacement directives I had in the root `go.mod` file. But now that there are no more replacement directives, none of them should be necessary.

Reply:
I'm talking about just the `go.mod` files in the `vendor` folder. Running `go build` and `go test` won't do anything to generate those. (I had to manually create them.)

Note that some of the vendored folders *do* have a `go.mod` file already. But some did not, so I added one for those packages. Take a look at the original PR to see what all I had to add: #41.